### PR TITLE
Fixes #271. Added server validation for 'None' action

### LINKED ISSUE #271: Should we catch empty batch actions?
Currently, when JS fails a button is displayed for the batch actions, which simply executes the selected action. 

However, when the 'Selected' is selected (so no action) and the button is clicked, the following error is thrown:
```
Undefined "attemptBatchNone" method. Does batch action "none" exist in your generator file?
```

This indicates that there is no server check for faulty requests here. We should probably check this in the controller and forward the user to the list page when this happens. 

Any opinions? @sescandell @loostro?

### PR DESCRIPTION
**Pull request by bobvandevijver**
See #271 for details on why.

### CODE REVIEW

File: Resources/translations/Admingenerator.fr.yml [hunk 1]
```diff
@@ -80,6 +80,8 @@ action.batch.delete:
     success:  Les enregistrements ont été supprimés avec succès.
     error:    Les enregistrements n'ont pu être supprimés.
 
+action.batch.none: Premièrement, vous devez sélectionner une action.
```
Comment (sescandell) [inline Resources/translations/Admingenerator.fr.yml]: French one would be: *"Merci de sélectionner une action"*

**Comment by sescandell** [comment]
:+1:

File: Resources/translations/Admingenerator.fr.yml (hunk 1 above)
Comment (bobvandevijver) [inline Resources/translations/Admingenerator.fr.yml]: Done!

**Comment by sescandell** [comment]
Thanks

**Pull request merged**